README.md

```diff
@@ -211,7 +211,7 @@ await wallet.open(representative);
 ### Send
 
 ```typescript
-send(address: string, amount: number | string): Promise<string>
+send(target: NanoTarget, amount: number | string): Promise<string>
```

Comment:
Explain somewhere what `NanoTarget` and `NanoAddress` are, people may be confused thinking it's a class.
And add an example.

You can also mention your new feature in the documentation, explaining that this library can resolve nano.to usernames !

Reply:
Also you can add your `NameService` to the doc, allowing people to resolve nano.to username with this lib.

Reply:
And finally you can check the feature in the roadmap :)

`- [x] Integrate [Nano name service](https://github.com/fwd/nano-to)`